go-controller/pkg/cni/bandwidth.go

```diff
@@ -87,11 +89,17 @@ func getPodBandwidth(ifname string) (ingressBPS, egressBPS int64, err error) {
 	}
 	// egressBPS
 	out, err = ovsGet("interface", ifname, "ingress_policing_rate", "")
-	if err == nil && len(out) > 0 {
-		egressBPS, err = strconv.ParseInt(out, 10, 64)
-		if err == nil {
-			egressBPS *= 1000
+	if err != nil {
+		return 0, 0, errors.Wrapf(err, "getPodBandwidth: failed to get ingress_policing_rate for interface %s", ifname)
```

Comment:
@fedepaol a couple error nits; remove the "getPodBandwidth: " here, and de-capitalize the "Failed" just below.

Reply:
Done, thanks